src/analytics/api/models/events.py

```diff
@@ -42,9 +49,10 @@ def from_bigquery(cls, tenant, sites, start_date, end_date, frequency, pollutant
         job_config = bigquery.QueryJobConfig()
         job_config.use_query_cache = True
 
-        query = client.query(QUERY, job_config)
+        dataframe = bigquery.Client().query(QUERY, job_config).result().to_dataframe()
+        dataframe.sort_values(["name", "datetime"], ascending=True, inplace=True)
 
-        return [dict(row) for row in query.result()]
+        return dataframe.to_dict(orient="records")
```

Comment:
@NoahNsimbe , please add some PR comments for reviewers to reference/appreciate either during or after review/merge duties are done. Helps with "onboarding".

Thanks!